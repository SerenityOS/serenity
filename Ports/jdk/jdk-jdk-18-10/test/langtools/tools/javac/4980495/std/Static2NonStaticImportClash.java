/*
 * @test  /nodynamiccopyright/
 * @bug 7101822 8133616
 * @summary Check the when clashing types are imported through an ordinary and static import,
 *          the compile-time error is properly reported.
 * @compile/fail/ref=Static2NonStaticImportClash.out -XDrawDiagnostics Static2NonStaticImportClash.java p1/A1.java p2/A2.java
 *
 */

import p2.A2.f;
import static p1.A1.f;

public class Static2NonStaticImportClash {
}
