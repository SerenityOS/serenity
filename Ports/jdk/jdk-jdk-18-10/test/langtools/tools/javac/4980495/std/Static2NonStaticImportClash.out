Static2NonStaticImportClash.java:11:1: compiler.err.already.defined.single.import: p2.A2.f
1 error
